REVIEWING.md

```diff
@@ -0,0 +1,274 @@
+    be tolerated for a number of very small PRs quick to review and thus merge.
+
+  - Production of PRs and Review of PRs should observe a simple ratio. For each
+    PR produced and merged, you should be participating in the process of
+    merging two other PRs as a reviewer. This will allow to produce as many PRs
+    as we merge, and keep the number of open PRs stable
+
+- Documentation can go a long way for reviewers and knowledge transfer:
+
+  - JSDoc for an API brings a lot to the  table, and eases the review
+
+  - Comments explaining complex algorithms or implementation choices can reduce
+    unneeded discussion on the minute details, as well as bring insight on the
+    said implementation choice for the future, and backtracking of issues and
+    their solutions
+
+  - Documenting how to use your code if not in the JSDoc (although this part
+    should be mandatory), can be done in some kind of markdown somewhere
+
+- Tests are important, if you add a feature, add the corresponding tests that
+  check the desired behaviour (and edge cases). You can define the scope of your
+  test with the scope of your feature/fix:
+
+  - Is the code valid by itself? -> unit tests
+
+  - Does it have negligible impact on code outside of itself? On the component
+    behaviour? -> Functional tests
+
+  - Has it a marginal impact on other components? -> Integration tests
+
+  - Is it a big change for everyone? -> End to end tests
+
+  - Never forget to write regression tests to ensure that we'll never trigger
+  bugs again
+
+  - Never forget to write compatibility tests to avoid break our stored data
+  formats and protocols
+
+- Tool-related habits:
+
+  - Linter:
+
+    - PRs should not add any lint/compilation warning
+
+    - Linter fixes should be done when an edited line is concerned by a lint
+      issue, do not go out of your way to fix things (Cleanup PRs are ok
+      though, to avoid an accumulation of warnings)
+
+  - GitHub:
+
+    - Per-commit diff view feature: It can be helpful to add new commits rather
+      than squash and rebase early-on, to facilitate incremental reviews for
+      your peers
+
+    - Reactions Emojis: Do not hesitate to express your support through those,
+      it can be a nice way to show support to others
+
+    - Review UI: Use the review UI to do one big review, and post it as once,
+    requesting changes, commenting or approving the PR. Please avoid requesting
+    changes for only minor things that are "optional", as it may prevent
+    merging.
+
+## Reviewing your peers, the gentle way
+
+We're all human beings, and we can't possibly know everything about every
+project, asking questions is perfectly fine when some piece of code seems a bit
+obscure. Take care not to block the process for a trivial question, that could
+easily be ignored.
+
+Giving an approval review should be incidental, not the standard. This doesn't
```

Comment:
I understand the meaning: it doesn't have to be a reflex to just approve some a PR. Define it as an `incident` is a bit too much in my opinion :)